lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -224,7 +224,7 @@ defmodule LivebookWeb.SessionLive.CellComponent do
 
       <%= if @cell_view.type == :elixir do %>
         <div class="absolute bottom-2 right-2">
-          <%= render_cell_status(@cell_view.validity_status, @cell_view.evaluation_status) %>
+          <%= render_cell_status(@cell_view, @cell_view.evaluation_status) %>
```

Comment:
Ah, I think we should pass the `evaluation_time_ms` as the third argument, rather than the whole `@cell_view`, so that LV doesn't need to re-render this part when an irrelevant part of `@cell_view` changes.